harold/plugins/deploy.py

```diff
@@ -226,6 +219,10 @@ def send_response(salons):
             request.write(json.dumps(salon_names))
             request.finish()
 
+        # The 'all' method returns a Twisted Deferred object.
```

Comment:
Just adding some extra explanation so that in the future someone knows why we're doing this `addCallback` magic here.